openapi/components/schemas/StorefrontQuoteReactivateOrder.yaml

```
@@ -0,0 +1,481 @@
+type: object
+required:
+  - websiteId
+  - customerId
+  - items
+properties:
+  id:
+    readOnly: true
+    description: ID of the quote.
+    type: string
+    maxLength: 50
+    example: qt_0YV7DES3WPC5J8JD8QTVNZBZNZ
+  type:
+    description: Specifies the type of the quote. A quote can be a subscription or a one-time purchase.
+    type: string
+    readOnly: true
+    enum:
+      - subscription-order
+      - one-time-order
+  action:
+    description: |-
+      Action of the quote for churned/canceled order reactivation.
+      On accept quote new invoice is issued to be and linked to `invoiceId`. Quote items are applied to the order.
+      The related order will have `pending` status.
+      Once the invoice is paid, the order becomes active on `settings.reactivate.effectiveTime` or immediately if `settings.reactivate.effectiveTime` is omitted.
+    type: string
+    enum:
+      - reactivate
+    example: reactivate
+  acceptanceConditions:
+    type: array
+    writeOnly: true
+    description: If conditions are fulfilled the quote is automatically accepted.
+    items:
+      type: string
+      enum:
+        - customer
+        - signature
+        - payment
+        - partial-payment
+      x-enumDescriptions:
+        customer: Quote can be accepted by customer.
+        signature: Quote can be accepted if merchant received the signed quote from the customer.
+        payment: Related invoice must be paid in full.
+        partial-payment: Related invoice must be paid in full or partially paid.
+    example: ['customer', 'payment']
+    default: ['customer']
+  acceptanceFulfillment:
+    type: array
+    readOnly: true
+    description: Conditions with fulfillment flag.
```

suggestion: to add more detail
I found this description difficult to understand (I'm not sure if my suggestion changes the meaning)

```suggestion
    description: |-
      Conditions that must be completed to fulfill the order. 
```